SteamCommand.cs

```diff
@@ -15,10 +15,12 @@ public static class SteamCommand {
       Thread.Sleep(100); // Slight delay for steam to become foreground
     }
 
-    public static void DownloadDepot(int depot, long manifest) {
-      Logging.Log($"download_depot {GAME_ID} {depot} {manifest}");
-      if (manifest == 0) return; // 0 indicates "No such manifest", so we shouldn't attempt to download it.
-      sim.Keyboard.TextEntry($"download_depot {GAME_ID} {depot} {manifest}");
+    public static void DownloadManifest(SteamManifest manifest) {
+      if (manifest is null) throw new System.ArgumentNullException(nameof(manifest));
```

Comment:
Don't throw exceptions, there's no exception handling. This will crash the app.
I suggest following the old idea, and simply electing to not download if there is no manifest.

Also, I'd prefer not using the `is` keyword, it's a bit too python-esque.

Reply:
Ah windows forms vs wpf must be different there, there's a window for uncaught exceptions in forms, so I figured it'd be fine to throw an exception that never happens anyway. Should I have it return if the data on the manifest is 0 too, like before?

Reply:
Even if there is a handler, I don't like showing a dialogue box that means nothing to the user.
I assumed you were replacing `ManifestData(0, 0, 0)` with `null`, so the `== 0` check is gone? If not, you should handle that case, too.

Reply:
In the cases where that used to be used I just don't include the manifests in the list, this was just a warning intelisense gave me once I got it working again.